### ISSUE TITLE
add dummy icons

### ISSUE THREAD

**Issue by onlygecko** [mardy/ttrss#6]
Hi,

first of all: Thanks for the last update! :-) 

I have another suggestion. Would it be possible to add dummy icons for feeds that have no icon? And here as well - a little bit of margin between the icon and the feed name would be nice. ;-) Example is attached. 

Benefits: 
- Consistent view in the feed overview
- easier to read

![screenshot20150727_211421020](https://cloud.githubusercontent.com/assets/3233376/8915405/1b616c2e-34a5-11e5-96fe-bd73c8f8fcaf.png)

**Comment by mardy** [comment]
Makes sense. Would you prefer just an empty icon or maybe a generic one like http://bazaar.launchpad.net/~ubuntu-art-pkg/ubuntu-themes/trunk/download/head:/emptysymbolic.svg-20140701092345-0bu0ug2hpoie2zmc-6/empty-symbolic.svg ?

**Comment by onlygecko** [comment]
If I had a wish free I would choose an rss symbol like this one: https://openclipart.org/detail/176721/simple-rss. It's seems to be a simple, discreet placeholder. What do You think?

I don't know if there's something like that in the ubuntu art repository.
